apps/nextjs/src/components/AppComponents/SectionContent/QuizSection/ImageAttribution/formatNumberRanges.test.ts

```diff
@@ -0,0 +1,52 @@
+import { formatNumberRanges } from "./formatNumberRanges";
+
+describe("formatNumberRanges", () => {
+  it("returns empty string for empty array", () => {
+    expect(formatNumberRanges([])).toBe("");
+  });
+
+  it("handles single number", () => {
+    expect(formatNumberRanges([5])).toBe("Q5");
+  });
+
+  it("handles consecutive numbers", () => {
+    expect(formatNumberRanges([1, 2, 3])).toBe("Q1-Q3");
+  });
+
+  it("handles non-consecutive numbers", () => {
+    expect(formatNumberRanges([1, 3, 5])).toBe("Q1,Q3,Q5");
```

Comment:
Should it not be "Q1, Q2, Q5"? (space after comma)

Reply:
Fair enough! I'll fix that now